content/en/user-manual/graphics/lighting/shadows.md

```diff
@@ -18,9 +18,23 @@ By default, shadow casting is disabled in PlayCanvas. You have to explicitly ena
 
 ![Model Component][6]
 
-Now you need to specify which graphical objects in your scene cast and receive shadows. By default, all model components receive shadows but do not cast them. Therefore, in order to see an entity cast shadows, select it in the Hierarchy, locate the model component in the Inspector and check the 'Cast Shadows' option.
+Now you need to specify which graphical objects in your scene cast and receive shadows. By default, all render and model components cast and receive shadows. To modify these properties, select the entity in the Hierarchy, locate the render or model component in the Inspector and uncheck the 'Cast Shadows' or 'Receive Shadows' option as required.
 
-Shadows should now be visible in the Editor's Viewport.
+## Shadow Cascades
+
+When a directional shadow is used over a large area, it often exhibits aliasing, where a shadow near the camera has a low resolution. Capturing the shadow in a single shadow map requires very high and impractical resolution to improve this.
+
+Shadow cascades help to fix this problem by splitting the camera view frustum along the viewing direction, and a separate shadow map is used for each split. This gives nearby objects one shadow map, and another shadow map captures everything in the distance, and optionally additional shadow maps in between.
+
+The following properties can be used to set up shadow cascades.
+
```

Comment:
Screenshots to compare with would be useful for users to tell the difference and know whether they should use it or not